documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1405,7 +1405,15 @@ If you include this property in the configuration, do not set the `collection.in
 
 |[[mongodb-property-snapshot-mode]]<<mongodb-property-snapshot-mode, `+snapshot.mode+`>>
 |`initial`
-|Specifies the criteria for running a snapshot upon startup of the connector. The default is *initial*, and specifies that the connector reads a snapshot when either no offset is found or if the change stream no longer contains the previous offset. The *never* option specifies that the connector should never use snapshots, instead the connector should proceed to tail the log.
+|Specifies the criteria for performing a snapshot when the connector starts.
+Set the property to one of the following values:
+
+`initial`::
+When the connector starts, if it does not detect a value in its offsets topic, it performs a snapshot of the database.
+
+`never`::
+When the connector starts, it does not perform a snapshot.
+Instead of running a snapshot, the connector tails the oplog, and emits `read` events for the most recently recorded transactions.
```

Comment:
@roldanbob This is not correct. In case of `never` no `read` events are emitted. Just the captured changes as the streaming is started and snapshotting skipped.